com.unity.render-pipelines.high-definition/Documentation~/Override-Cloud-Layer.md

```diff
@@ -11,13 +11,19 @@ The **Cloud Layer** uses the [Volume](Volumes.html) framework, so to enable and
 
 After you add a **Cloud Layer** override, you must enable it in the override itself. In the override, Check the **Enable** property. HDRP now renders **Cloud Layer** for any Camera this Volume affects.
 
+<a name="DefaultCloudMap"></a>
+
+## Using the default Cloud Map
+
+An example Cloud Map texture named `DefaultCloudLayer` is included in the HDRP package.
+As it is a CustomRenderTexture, you can duplicate the asset in your project and tweak some parameters. To do so, first duplicate the `DefaultCloudLayer` CustomRenderTexture. Then duplicate the `DefaultCloudLayerMaterial` and select it in the inspector of your CustomRenderTexture.
```

Comment:
This example Cloud Map is a read-only CustomRenderTexture. This means that, if you want to modify any of its properties, you need to:
1. Duplicate the **DefaultCloudLayer** asset.
2. Duplicate the Material the **DefaultCloudLayer** asset references. This Material is called **DefaultCloudLayerMaterial**.
3. In the Inspector for the new **DefaultCloudLayer**, assign the new **DefaultCloudLayerMaterial** to the **Material** property.